Packs/AWS-SecurityHub/Integrations/AWSSecurityHubEventCollector/AWSSecurityHubEventCollector.py

```diff
@@ -17,29 +16,72 @@
 PRODUCT = "Security Hub"
 TIME_FIELD = "CreatedAt"
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+DATETIME_FORMAT_NO_MS = "%Y-%m-%dT%H:%M:%SZ"  # Fallback format without milliseconds
 DEFAULT_FIRST_FETCH = "3 days"
 DEFAULT_MAX_RESULTS = 1000
 API_MAX_PAGE_SIZE = 100  # The API only allows a maximum of 100 results per request. Using more raises an error.
+RETRY_LIMIT_INCREMENT = 100  # How much to increase limit when AWS returns all events as duplicates
+MAX_AWS_LIMIT = 10000  # Maximum limit to prevent excessive API calls
```

Comment:
Why did we decide to go with incremental limit raising instead of documenting this limitation as we do in other collectors?

Reply:
I think this is a better approach.